bin/labelcat.js

```diff
@@ -1,24 +1,59 @@
 #!/usr/bin/env node
 
 const util = require('../src/util.js');
+const settings = require('../settings.json');
 
 require(`yargs`)
   .demand(1)
   .command(
     `retrieveIssues <repoDataFilePath> <issuesDataFilePath>`,
     `Retrieves issues from a .txt file of gitHub repositories.`,
     {},
-    async (opts) => {
+    async opts => {
       util.makeCSV(
         await util.retrieveIssues(opts.repoDataFilePath),
         opts.issuesDataFilePath
       );
     }
   )
+  .command(
+    `createDataset <datasetName> [multilabel]`,
```

Comment:
Make `multilabel` a flag instead of an argument, that way yargs can parse it as a boolean. Users would just need to add `--multilabel` to set the value of the flag to true.